src/web/mjs/engine/DiscordPresence.mjs

```diff
@@ -101,14 +101,16 @@ export default class DiscordPresence {
         }
     }
 
-    stopDiscordPresence() {
+    stopDiscordPresence(reset = false) {
```

Comment:
May split into two methods instead of introducing flag parameters, e.g. `stopDiscordPresenceGracefully()` and `terminateDiscordPresence()` or similar